GitHubRepositories.ps1

```diff
@@ -1,6 +1,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+$script:GitHubRepositoryTypeName = 'GitHub.Repository'
+$script:GitHubRepositoryTopicTypeName = 'GitHub.RepositoryTopic'
+$script:GitHubRepositoryContributorTypeName = 'GitHub.RepositoryContributor'
+$script:GitHubRepositoryCollaboratorTypeName = 'GitHub.RepositoryCollaborator'
+$script:GitHubRepositoryLanguageTypeName = 'GitHub.RepositoryLanguage'
+$script:GitHubRepositoryTagTypeName = 'GitHub.RepositoryTag'
```

Comment:
Might be worthwhile to initialize these as read-only script vars the way we do in GitHubCore:

```suggestion
@{
    GitHubRepositoryTypeName = 'GitHub.Repository'
    GitHubRepositoryTopicTypeName = 'GitHub.RepositoryTopic'
    GitHubRepositoryContributorTypeName = 'GitHub.RepositoryContributor'
    GitHubRepositoryCollaboratorTypeName = 'GitHub.RepositoryCollaborator'
    GitHubRepositoryLanguageTypeName = 'GitHub.RepositoryLanguage'
    GitHubRepositoryTagTypeName = 'GitHub.RepositoryTag'
 }.GetEnumerator() | ForEach-Object {
     Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
 }
```

Reply:
Done.

As an aside, this repo has `alignPropertyValuePairs` as `false` in its vscode settings. Personally, I think they are much easier to read when aligned i.e.:

```powershell
@{
    GitHubRepositoryTypeName             = 'GitHub.Repository'
    GitHubRepositoryTopicTypeName        = 'GitHub.RepositoryTopic'
    GitHubRepositoryContributorTypeName  = 'GitHub.RepositoryContributor'
    GitHubRepositoryCollaboratorTypeName = 'GitHub.RepositoryCollaborator'
    GitHubRepositoryLanguageTypeName     = 'GitHub.RepositoryLanguage'
    GitHubRepositoryTagTypeName          = 'GitHub.RepositoryTag'
 
}.GetEnumerator() | ForEach-Object {
    Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
}
```
```

Reply:
I've been on both sides of that argument over the years at Microsoft on different teams.  I'm currently on the side of _not_ doing that.  I _totally_ agree that in isolation, that's easier to read.  It mostly comes down to maintenance.  If it's likely that values are to be added/removed over time, then it becomes annoying both from a maintainer and code reviewer when having to deal with re-adjusting the spacing within there as things are added and removed.  And even if _this_ one is likely to remain more-or-less static, it then adds precedent/pressure to do so in other parts of the code that may be less static, or the code then starts to lack conformity.

Reply:
That's the beauty of modern editors like VSCode, Shift-Alt-F and it formats it for you!